now.json

```diff
@@ -1,6 +1,6 @@
 {
   "redirects": [
-    {"source": "/lessons", "destination": "/s"},
-    {"source": "/courses", "destination": "/s"}
+    {"source": "/lessons", "destination": "/s?type=lesson"},
```

Comment:
This is starting to remind me of how codesandbox does links:

http://codesandbox.io/s/vanilla
http://codesandbox.io/s/github
http://codesandbox.io/s/vue

etc, etc, etc.